client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -166,17 +201,30 @@ function AddTestToQueueWithConfirmation({
                 disabled={disabled}
                 variant="secondary"
                 onClick={async () => {
-                    if (hasAutomationSupport) {
-                        setShowConfirmation(true);
+                    const conflictingReportExists =
+                        existingTestPlanReports.some(report => {
```

Comment:
`existingTestPlanReports` is potentially falsey, so we should use optional chaining here, too. It might also be good to perform this calculation outside the handler just like we do for `hasAutomationSupport`.